arcgis-ios-sdk-samples/Layers/Show labels on layers/ShowLabelsOnLayersViewController.swift

```diff
@@ -16,90 +16,63 @@ import UIKit
 import ArcGIS
 
 class ShowLabelsOnLayersViewController: UIViewController {
-    @IBOutlet private weak var mapView: AGSMapView!
+    @IBOutlet private weak var mapView: AGSMapView! {
+        didSet {
+            // Create a map with a light gray canvas basemap.
+            mapView.map = AGSMap(basemapStyle: .arcGISLightGrayBase)
+            // Set the map viewpoint to show the layer.
+            mapView.setViewpointCenter(AGSPoint(x: -10840000, y: 4680000, spatialReference: .webMercator()), scale: 20000000)
+        }
+    }
     
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-        // add the source code button item to the right of navigation bar
-        (navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["ShowLabelsOnLayersViewController"]
-        
-        // create a map with a light gray canvas basemap.
-        let map = AGSMap(basemapStyle: .arcGISLightGrayBase)
-        // assign the map to the map view
-        mapView.map = map
-        
-        /// A URL for a feature service layer.
+    /// Adds a feature layer and its labels.
+    private func addLayerAndLabels() {
```

Comment:
You might consider turning this into a `makeMap()` function, since all it is doing is configuring part of the map. You might also make another method, like `makeCongressionalDistrictsLayer()` or something similar, that creates and configures the feature layer.

Reply:
I added a `makeMap()` function. Please let me know what you think!